opensrp-app/src/test/java/org/smartregister/sync/intent/SettingsSyncIntentServiceTest.java

```diff
@@ -27,11 +33,18 @@ public void setUp() throws Exception {
                 .get();
     }
 
-    @Ignore
+    @After
+    public void tearDown() throws Exception {
+        ReflectionHelpers.setStaticField(Context.class, "context", null);
+    }
+
     @Test
     public void onHandleIntent() {
```

Comment:
Could you update the test name to march the convention.

Reply:
Cool